pg_read_uncommitted.c

```diff
@@ -78,9 +82,15 @@ pgru_ExecutorStart (QueryDesc *queryDesc, int eflags)
 	if (pgru_enabled && IsMVCCSnapshot(queryDesc->snapshot))
 	{
 		if (pgru_show_deleted)
+#if PG_VERSION_NUM >= 120000
+			queryDesc->snapshot->snapshot_type = SNAPSHOT_ANY;
+		else
+			queryDesc->snapshot->snapshot_type = SNAPSHOT_DIRTY;
+#else
 			queryDesc->snapshot->satisfies = HeapTupleSatisfiesAny;
 		else
 			queryDesc->snapshot->satisfies = HeapTupleSatisfiesDirty;
+#endif
```

Comment:
I think it'd be more readable to have the `#if ... #else ... #endif` specified twice rather than having two `else` branches with only one optionally compiled.

Reply:
Done in 2cd163e

Reply:
Thanks!